catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -172,6 +174,7 @@ public Integer getReadLockIntervalMilliseconds() {
    * called whenever an existing route is updated.
    */
   public void init() {
+    LOGGER.debug("Configuring monitor for {}", monitoredDirectory);
```

Comment:
Maybe make this "Initializing monitor for..." to distinguish between the init() and configure() method below.